pygfx/renderers/wgpu/_renderer.py

```diff
@@ -266,6 +287,21 @@ def blend_mode(self, value):
         if isinstance(self._target, wgpu.gui.WgpuCanvasBase):
             self._target.request_draw()
 
+    @property
+    def sort_objects(self):
+        """Whether to sort world objects before rendering.
+
+        * ``True``: the render order is defined by 1) the object's distance to the
+          camera; 2) the object's ``render_order`` property; 3) the position object
+          in the scene graph (based on a depth-first search).
```

Comment:
Could you reorder the text in the docstring so that it aligns with the code? 2 comes before 1.

Reply:
Oh it does indeed!